Vagrantfile

```diff
@@ -74,6 +74,12 @@ sudo chsh -s /bin/zsh vagrant
 # create the virualenv for vagrant
 sudo -u vagrant virtualenv -p python3 /home/vagrant/venv
 
+# keep node modules on vagrant instance only
+# releases stress on webpack watch, keeps stuff smoothly separated
+sudo -u vagrant mkdir -p /home/vagrant/node_modules
+sudo -u vagrant mkdir -p /vagrant/node_modules
+sudo mount --bind /home/vagrant/node_modules /vagrant/node_modules
```

Comment:
Okay, neat. @homu r+

On Tue, Mar 29, 2016 at 11:03 PM Benjamin Kampmann <notifications@github.com>
wrote:

> In Vagrantfile
> <https://github.com/beavyHQ/beavy/pull/94#discussion_r57800199>:
>
> > @@ -74,6 +74,12 @@ sudo chsh -s /bin/zsh vagrant
> >  # create the virualenv for vagrant
> >  sudo -u vagrant virtualenv -p python3 /home/vagrant/venv
> >
> > +# keep node modules on vagrant instance only
> > +# releases stress on webpack watch, keeps stuff smoothly separated
> > +sudo -u vagrant mkdir -p /home/vagrant/node_modules
> > +sudo -u vagrant mkdir -p /vagrant/node_modules
> > +sudo mount --bind /home/vagrant/node_modules /vagrant/node_modules
>
> that's exactly it. and that takes them off the NFS – therefore faster for
> webpack-watch!
>
> —
> You are receiving this because you were assigned.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/beavyHQ/beavy/pull/94/files/0c96844d48b73fbddf885bc13e18e37f97e253d9#r57800199>
>